runtime-builder/tests/integration/go-test.yaml

```diff
@@ -1,7 +1,6 @@
 steps:
 - name: gcr.io/gcp-runtimes/integration_test
   args: [
-    '--no-deploy',
```

Comment:
`--no-deploy` used to explicitly tell the integration testing framework not to deploy the sample app. now, it's implied if you pass the `--url` flag